projects/netgrif-components-core/src/assets/i18n/en.json

```diff
@@ -232,6 +232,8 @@
             "phone": "Entered text must be in telephone number format",
             "email": "Entered text must be in email format",
             "dateRange": "Entered date must be between {{left}} and {{right}}",
+            "datePast": "Entered date must be within {{right}}",
+            "dateFuture": "Entered date must be from {{left}}",
```

Comment:
I believe `... must be earlier than/later than` would be much clearer

Reply:
done